horovod/spark/torch/remote.py

```diff
@@ -36,9 +36,12 @@
 
 
 def RemoteTrainer(estimator, metadata, last_checkpoint_state, run_id, dataset_idx):
+    # Utility functions
+    make_list_if_not_list = make_list_if_not_list_fn()
+
     # Estimator parameters
-    loss_fns_pre_train = estimator.getLoss()
-    loss_constructors = estimator.getLossConstructors()
+    loss_fns_pre_train = make_list_if_not_list(estimator.getLoss())
```

Comment:
We have tests in `test_spark_torch.py` that do not provide loss as a list, but the tests are passing.  Based on this, I'm not sure how that's possible, do you?

Reply:
In the init function of the TorchEstimator, we had a check if loss and loss_constructors were a list, and if not, we made them a list. This problem came up when we use est.setLoss().